app/src/main/res/values-eo/strings.xml

```diff
@@ -544,121 +544,121 @@
-
-    <string name="configLabel_alarms_snooze">Snooze for</string>  <!-- TODO -->
-    <string name="configLabel_alarms_snooze_summary"><xliff:g id="timePeriod" example="10 minutes">%s</xliff:g></string>  <!-- TODO -->
-
-    <string name="configLabel_alarms_upcoming">Reminder within</string>  <!-- TODO -->
-    <string name="configLabel_alarms_upcoming_summary"><xliff:g id="timePeriod" example="10 minutes">%s</xliff:g></string>  <!-- TODO -->
-
-    <string name="configLabel_alarms_hardwarebuttons">Hardware Buttons</string>  <!-- TODO -->
-    <string name="configLabel_alarms_hardwarebuttons_summary"><xliff:g id="action" example="Snooze">%s</xliff:g></string>  <!-- TODO -->
+    <string name="configLabel_alarmClock">Alarmoj</string>            <!-- group title -->
+    <string name="configLabel_alarmClock_emptyMsg">Neniuj alarmoj</string>     <!-- a label that is displayed in place of an empty listview -->
+    <string name="configAction_addAlarm">Agordi alarmon</string>               <!-- button (contentDescription) -->
+    <string name="configAction_addNotification">Agordi sciigon</string> <!-- button (contentDescription) -->
+    <string name="configAction_deleteAlarm">Forigi ĉiujn</string>               <!-- button (contentDescription) -->
+    <string name="configAction_clearAlarms">Forgi</string>                <!-- action button / menu item -->
+    <string name="configAction_clearOffset">Forviŝi</string>                <!-- action button / menu item -->
+    <string name="configAction_setAlarmOffset">Agordi deŝovon</string>         <!-- button (contentDescription) -->
+    <string name="configAction_setAlarmType">Agordi tipon</string>         <!-- button (contentDescription) -->
+    <string name="configAction_setAlarmLabel">Agordi etikedon</string>         <!-- button (contentDescription) -->
+    <string name="configAction_setAlarmLocation">Agordi pozicion</string>         <!-- button (contentDescription) -->
+    <string name="configAction_setAlarmRepeat">Agordi ripeton</string>         <!-- button (contentDescription) -->
+    <string name="configAction_setAlarmEvent">Agordi okazaĵon</string>         <!-- button (contentDescription) -->
+    <string name="configAction_setAlarmTime">Agordi konstantan tempon</string>         <!-- button (contentDescription) -->
+    <string name="configAction_setAlarmSound">Agordi sonon</string>         <!-- button (contentDescription) -->
+    <string name="configAction_alarmMenu">Alarma menuo</string>         <!-- button (contentDescription) -->
+
+    <string name="alarmAction_dismiss">Rezigni</string>                <!-- notification action button -->
+    <string name="alarmAction_snooze">Prokrasti</string>                  <!-- notification action button -->
+    <string name="alarmAction_timeout">Silentiĝis</string>                  <!-- notification action button -->  <!-- TODO -->
```

Comment:
This isn't used anywhere (so actually safe to remove) - at one point there was an actual "Timeout" button but it was replaced with better UI. 

It could be used as a notification title, or as a contentDescription, but should be applicable to a button (verb).